src/Microsoft.VisualStudio.ProjectSystem.Managed/GlobalSuppressions.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+// This file is used by Code Analysis to maintain SuppressMessage 
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given 
+// a specific target and scoped to a namespace, type, member, etc.
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2213:Disposable fields should be disposed", Justification = "<Pending>", Scope = "member", Target = "~F:Microsoft.VisualStudio.ProjectSystem.LanguageServices.WorkspaceContextHost.WorkspaceContextHostInstance._subscriptions")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2213:Disposable fields should be disposed", Justification = "<Pending>", Scope = "member", Target = "~F:Microsoft.VisualStudio.ProjectSystem.LanguageServices.WorkspaceContextHost.WorkspaceContextHostInstance._applyChangesToWorkspaceContext")]
```

Comment:
Every single usage of OnceDisposedOnceInitialized/OnceDisposedOnceInitializedAsync is going to fire here, this is only noise in this repro because of this. We should really just turn the rule off.

Reply:
I'm OK with just turning off the rule.  If we have it on we should enforce it.  If it is not useful then turn it off.

Reply:
I'll do that in a followup PR.

Reply:
👍